### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by edavidj** [OpenLiberty/draft-guide-jax-rs-reactive-client#50]
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

Comments:
- Time: 15 min
- In the section just after running `startContainer.sh` there's the line `Switching to an reactive programming model has freed up`, which should be `Switching to a reactive programming model...`
- Early on in the "What is reactive programming" section the line `One of the greatest advantage of reactive programming ...` should become `One of the greatest advantages of reactive programming...`
- Didn't run into any technical or logical errors, content was clear to understand.

**Comment by TiagoF99** [comment]
- All feedback has been addressed and can be verified here: https://draft-guides.mybluemix.net/guides/jax-rs-reactive-client.html
